### DISCUSSION TITLE
How can i create an offline server

### DISCUSSION THREAD

**Discussion by ltxhhz** [OscarNOW/minecraft-server#145] [Questions] [answered]
I can't find any configurations for this.

**Comment by OscarNOW** [comment, marked as answer]
Offline servers are currently not supported with this package. You could look at the minecraft-protocol package that does support it (but you'll have to work more closely with the protocol language) or you could fork this repo and relatively easily add support for offline servers yourself.

**Reply by ltxhhz** [reply]
i see,thanks for your reply
